### ISSUE TITLE
NoIP now uses captcha

### ISSUE THREAD

**Issue by slnc** [loblab/noip-renew#96]
noip now uses recaptcha on its login page which breaks any automatic way of refreshing hosts.

**Comment by ffich** [comment]
In fact the script is not working anymore... it's returning "No hosts or host table rows not found".
Is a possible outcome of the mentioned issue? No solution at all?

Thanks 
F.

**Comment by slnc** [comment]
I've been on duckdns.org (free) since I filed the bug and no problems so far.

**Comment by ffich** [comment]
You mean with this same script only changing the ip of the dynamic host provider?

**Comment by neothematrix** [comment]
check my fork, it's being kept up to date and currently working: https://github.com/neothematrix/noip-renew/

**Comment by slnc** [comment]
> You mean with this same script only changing the ip of the dynamic host provider?

No, I meant not using this script anymore. Updating a dynamic host via duckdns is just a wget call.

**Comment by abir1909** [comment]
> check my fork, it's being kept up to date and currently working: https://github.com/neothematrix/noip-renew/
I tried it my RPI and it says username or password incorrect. Is it still working?
